kivy/uix/video.py

```diff
@@ -130,13 +171,21 @@ class Video(Image):
     _video_load_event = None
 
     def __init__(self, **kwargs):
+        # B/C case source is given. use it as video source
+        if 'source' in kwargs:
+            kwargs['video_source'] = kwargs['source']
+        # Case preview is given
+        if 'preview_source' in kwargs:
+            kwargs['source'] = kwargs['preview_source']
+
```

Comment:
```suggestion
        # Case preview is given
        if kwargs.get('preview_source'):
            kwargs['source'] = kwargs['preview_source']

```

I think it's better if we don't override `video_source`, but leave it to whatever the user set it to. And instead use the appropriate value later.

Also, we need to be careful not to overwrite in case `preview_source` is empty.